espresso/dockerfiles/l1-geth/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# L1 Geth Dockerfile, simplified from ops/docker/deployment-utils/Dockerfile
+FROM debian:12.7-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install runtime dependencies
+RUN apt-get update && apt-get install -y \
+    curl \
+    jq \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Geth
+RUN ARCH=$(dpkg --print-architecture) && \
+    echo "Detected architecture: $ARCH" && \
+    case "$ARCH" in \
+        "amd64") \
+            GETH_URL="https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.15.11-36b2371c.tar.gz" && \
+            GETH_SHA="a14a4285daedf75ea04a7a298e6caa48d566a2786c93fc5e86ec2c5998c92455" && \
+            GETH_DIR="geth-linux-amd64-1.15.11-36b2371c" && \
+            VERIFY_SHA="true" \
+            ;; \
+        "arm64") \
+            GETH_URL="https://gethstore.blob.core.windows.net/builds/geth-linux-arm64-1.15.11-36b2371c.tar.gz" && \
+            GETH_SHA="148ec84db2268fa846ae68f6445f0c98d33e95069e40fe8c74b43ea5eb53df7b" && \
+            GETH_DIR="geth-linux-arm64-1.15.11-36b2371c" && \
+            VERIFY_SHA="true" \
+            ;; \
+        *) \
+            echo "Unsupported architecture: $ARCH" && exit 1 \
+            ;; \
+    esac && \
+    echo "Downloading: $GETH_URL" && \
+    curl -L "$GETH_URL" -o geth.tar.gz && \
+    tar -xvf geth.tar.gz && \
+    mv "$GETH_DIR/geth" /usr/local/bin/geth && \
+    rm -rf geth.tar.gz "$GETH_DIR" && \
+    chmod +x /usr/local/bin/geth
+
```

Comment:
We should check the hash GETH_SHA here.

Reply:
Oh, we do! It's located around 10+ lines above, the command starting from `GETH_SHA=`.

Reply:
Fixed in https://github.com/EspressoSystems/optimism-espresso-integration/pull/182/commits/5559716cc444c53b6b00fecdc0a0eff077b03ca7, with a `sha256sum` check.